lib/RocketMap.php

```diff
@@ -4,8 +4,16 @@
 
 class RocketMap extends Scanner
 {
-    public function get_active($eids, $swLat, $swLng, $neLat, $neLng, $tstamp = 0, $oSwLat = 0, $oSwLng = 0, $oNeLat = 0, $oNeLng = 0)
+    private $cp_multiplier;
+    public function __construct()
     {
+        parent::__construct();
+        $this->set_cp_multiplier();
+    }
+
+    public function get_active($eids, $miniv, $minlevel, $exminiv, $swLat, $swLng, $neLat, $neLng, $tstamp = 0, $oSwLat = 0, $oSwLng = 0, $oNeLat = 0, $oNeLng = 0)
```

Comment:
As RocketMap_Sloppy has not been updated, this throws the following warning:
```
PHP Warning:  Declaration of Scanner\RocketMap_Sloppy::get_active($eids, $swLat, $swLng, $neLat, $neLng, $tstamp = 0, $oSwLat = 0, $oSwLng = 0, $oNeLat = 0, $oNeLng = 0) should be compatible with Scanner\RocketMap::get_active($eids, $miniv, $minlevel, $exminiv, $swLat, $swLng, $neLat, $neLng, $tstamp = 0, $oSwLat = 0, $oSwLng = 0, $oNeLat = 0, $oNeLng = 0) 
```
I'd also request swapping to camelCase to keep in the styling of the other vars (change `$minlevel` to `$minLevel` for example). I understand they aren't all in camelCase but going forward we should be doing this, it also makes reading the variables a little easier :)

Reply:
I cannot get the rocketmaps sloppy  error to appear. I dont use rocketmaps sloppy nor do i have a database for it. I have display_errors on and and Display all errors and still nothing appear with either Monocle or Rocketmaps. My guess is it's missing the constructor but thats just a guess from the code and no testing. I'll take care of the camel casing.

Reply:
That's strange, because all libraries are loaded by default. The reason it throws the error is because `get_active()` for RocketMap_Sloppy has a different list of arguments when compared to RocketMap (because you've added `$miniv, $minlevel, $exminiv,`).

I'll take another look in the morning and see if this is not just something exclusive to my test build.

Reply:
Thats weird because rocketmap_sloppy is basically an empty object in all sense other than it inherits rocketmaps. I had one other guy on rocketmaps get the same error but other than that other rocketmaps instances were running fine.